src/script/page/LeftSidebar/panels/StartUI/StartUI.tsx

```diff
@@ -43,6 +43,7 @@ import {Conversation} from 'src/script/entity/Conversation';
 import {ServiceEntity} from 'src/script/integration/ServiceEntity';
 import {showUserModal} from '../../../../page/Modals/UserModal/UserModal';
 import {showServiceModal} from '../../../../page/Modals/ServiceModal/ServiceModal';
+import {showInviteModal} from '../../../../page/Modals/InviteModal/InviteModal';
```

Comment:
Can you try adding a `index.ts` at the root of `InviteModal` that just exposes the API, so that we can import it without having to do `InviteModal/InviteModal`?

Reply:
Actually, we already have an alias for `Components` maybe it's a good idea to move all the modals to `Components/Modals` :thinking: 

Still adding an `index.ts` at the root of the folder is good because you then import by doing
```js
import {Component} from 'Components/Modal/Component`
```
instead of
```js
import {Component} from 'Components/Modal/Component/Component`
```